.github/workflows/pull-request.yml

```diff
@@ -24,41 +28,49 @@ jobs:
               env:
                   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-            - name: Create error comment
-              # When the previous steps fails, the workflow would stop. By adding this
-              # condition you can continue the execution with the populated error message.
-              if: always() && (steps.lint_pr_title.outputs.error_message != null)
-              # https://github.com/marocchino/sticky-pull-request-comment
-              uses: marocchino/sticky-pull-request-comment@v2
-              with:
-                  header: pr-title-lint-error
-                  message: |
-                      **Hi there @${{ github.event.pull_request.user.login }}** 👋,
+            - run: echo "status=failure" >> $GITHUB_OUTPUT
+              id: failure
+              if: failure()
+            - run: echo "status=success" >> $GITHUB_OUTPUT
+              id: success
+              if: success()
 
-                      and **THANK YOU** from the bottom of our hearts for opening this Pull Request! ❤️
+            - run: echo "status=failure" >> $GITHUB_OUTPUT
```

Comment:
Why is there this block 2 times?

Reply:
Because there's no way to check the "status" of specific step in specific job. Is a missing feature in GitHub Actions.

I need to create an output in order to trigger condition properly in the `error-message` workflow

Reply:
Solved by [1ab72cd](https://github.com/rust-cleaners/polished-css/pull/36/commits/1ab72cd889e76d28bd1903ae55171cd365e3ecb5)